src/view/pages/BookingPage.php

```diff
@@ -22,7 +23,23 @@
         </div>
     </div>
     ";
-    echo date('H:i', $_SESSION['activeBooking']['expiry']);
+}
+// If the active booking has expired
+else if (isset($_SESSION['activeBooking']) && $_SESSION['activeBooking']['expiry'] <= time()) {
+    // The proceed button is hidden by as it has to be included otherwise the close button will not work
```

Comment:
It took 30 minutes to figure out and still don't understand why, as the two modals are basicially the same. BS language